engine/table/src/main/java/io/deephaven/engine/table/impl/updateby/emstd/BigIntegerEmStdOperator.java

```diff
@@ -0,0 +1,146 @@
+package io.deephaven.engine.table.impl.updateby.emstd;
+
+import io.deephaven.api.updateby.OperationControl;
+import io.deephaven.chunk.Chunk;
+import io.deephaven.chunk.LongChunk;
+import io.deephaven.chunk.attributes.Values;
+import io.deephaven.engine.rowset.RowSequence;
+import io.deephaven.engine.table.ColumnSource;
+import io.deephaven.engine.table.MatchPair;
+import io.deephaven.engine.table.impl.updateby.UpdateByOperator;
+import io.deephaven.engine.table.impl.util.RowRedirection;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import java.math.MathContext;
+
+import static io.deephaven.util.QueryConstants.NULL_LONG;
+
+public class BigIntegerEmStdOperator extends BaseBigNumberEmStdOperator<BigInteger> {
+    public class Context extends BaseBigNumberEmStdOperator<BigInteger>.Context {
+        protected Context(final int affectedChunkSize, final int influencerChunkSize) {
+            super(affectedChunkSize);
+        }
+
+        @Override
+        public void accumulateCumulative(@NotNull final RowSequence inputKeys,
+                                         @NotNull final Chunk<? extends Values>[] valueChunkArr,
+                                         @Nullable final LongChunk<? extends Values> tsChunk,
+                                         final int len) {
+            setValueChunks(valueChunkArr);
+
+            // chunk processing
+            if (timestampColumnName == null) {
+                // compute with ticks
+                for (int ii = 0; ii < len; ii++) {
+                    // read the value from the values chunk
+                    final BigInteger input = objectValueChunk.get(ii);
+                    if (input == null) {
+                        handleBadData(this, true);
+                    } else {
+                        final BigDecimal decInput = new BigDecimal(input);
+                        if (curEma == null) {
+                            curEma = decInput;
+                            curVariance = BigDecimal.ZERO;
+                            curVal = null;
+                        } else {
+                            //  incremental variance = alpha * (prevVariance + (1 − alpha) * (x − prevEma)^2)
```

Comment:
For these equations, I think it is worth putting a note with the pdf URL and title.  This applies to all files.